extensions/wikia/WhereIsExtension/templates/list.tmpl.php

```diff
@@ -138,7 +149,7 @@
 
 	$.loadJQueryAutocomplete(function() {
 		$('#wikiSelectTagName').autocomplete({
-			serviceUrl: wgServer+wgScript+'?action=ajax&rs=WikiFactoryTags::axQuery',
+			serviceUrl: mw.config.get( 'wgServer' ) + mw.config.get( 'wgScript' ) + encodeURIComponent('?action=ajax&rs=WikiFactoryTags::axQuery'),
```

Comment:
`encodeURIComponent` here will break the URL as it will encode the `?` and so forth. It's not needed anyway.